fnal_column_analysis_tools/processor/executor.py

```diff
@@ -74,14 +74,21 @@ def _work_function(item):
 @lru_cache(maxsize=128)
 def _get_chunking(filelist, treename, chunksize):
     items = []
-    for fn in filelist:
-        nentries = uproot.numentries(fn, treename)
+    executor = None if len(filelist) < 5 else concurrent.futures.ThreadPoolExecutor(10)
```

Comment:
I was going to ask if this hardcoded 10 thread option could be configured via module global or something.

Reply:
By "module global" is there already some kind of configuration file somewhere that you could point me to?